src/rag_processor/core/sentry.py

```
@@ -206,7 +214,9 @@ def before_send_hook(event: dict[str, Any], hint: dict[str, Any]) -> dict[str, A
     return event
 
 
-def before_breadcrumb_hook(crumb: dict[str, Any], hint: dict[str, Any]) -> dict[str, Any] | None:
+def before_breadcrumb_hook(
+    crumb: dict[str, Any], hint: dict[str, Any]
```

Ruff (0.14.7)

Unused function argument: `hint`

(ARG001)